src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -43,6 +43,20 @@
 
 logger = getLogger(__name__)
 
+# We allow some common aliases to be used
+OPTION_ALIASES = {
+    "HEADER": ("SKIP_HEADER", lambda val: 1 if val else 0),
+    "DELIMITER": ("FIELD_DELIMITER", lambda val: val),
+    "SEP": ("FIELD_DELIMITER", lambda val: val),
```

Comment:
both will map to `FIELD_DELIMITER`? I wonder how this mapping is determined. Are those aliases popular/required somewhere?

Reply:
This comes from usages like these: https://sparkbyexamples.com/spark/spark-read-options/?expand_article=1